administration/__generated__/globalTypes.ts

```diff
@@ -7,6 +7,11 @@
 // START Enums and Input Objects
 //==============================================================
 
+export interface AuthDataInput {
+  email: string;
+  password: string;
```

Comment:
Should this be a passwordHash?

Reply:
Else we are transmitting plain text passwords to the server.

Reply:
I created an issue. https://github.com/ehrenamtskarte/ehrenamtskarte/issues/322